roles/nginx/defaults/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
-logs_root: /var/log/nginx
+nginx_logs_root: /var/log/nginx
 nginx_user: www-data
 strip_www: true
+nginx_fastcgi_buffers: 8 8k
```

Comment:
Add a comment with a link to https://rtcamp.com/tutorials/nginx/tweaking-fastcgi-buffers/ ?